examples/src/main/java/io/dapr/examples/configuration/grpc/ConfigurationClient.java

```diff
@@ -106,35 +111,83 @@ public static void getConfigurations(DaprPreviewClient client) {
    * @param client DaprPreviewClient object
    */
   public static void subscribeConfigurationRequestWithSubscribe(DaprPreviewClient client) {
-    System.out.println("*****Subscribing to keys using subscribe method: " + keys.toString() + " *****");
+    System.out.println("Subscribing to key: myconfig1");
     AtomicReference<Disposable> disposableAtomicReference = new AtomicReference<>();
-    SubscribeConfigurationRequest req = new SubscribeConfigurationRequest(CONFIG_STORE_NAME, keys);
+    SubscribeConfigurationRequest req = new SubscribeConfigurationRequest(
+        CONFIG_STORE_NAME, Collections.singletonList("myconfig1"));
     Runnable subscribeTask = () -> {
-      Flux<List<ConfigurationItem>> outFlux = client.subscribeToConfiguration(req);
+      Flux<SubscribeConfigurationResponse> outFlux = client.subscribeToConfiguration(req);
       disposableAtomicReference.set(outFlux
           .subscribe(
-              cis -> cis.forEach(ConfigurationClient::print)
+              cis -> {
+                cis.getItems().forEach(ConfigurationClient::print);
+              }
           ));
     };
     new Thread(subscribeTask).start();
-    try {
-      // To ensure that subscribeThread gets scheduled
-      Thread.sleep(0);
-    } catch (InterruptedException e) {
-      e.printStackTrace();
-    }
+
+    // To ensure that subscribeThread gets scheduled
+    inducingSleepTime(0);
+
     Runnable updateKeys = () -> {
       int i = 1;
       while (i <= 3) {
-        executeDockerCommand(i);
+        executeDockerCommand("myconfig1", i);
         i++;
       }
     };
     new Thread(updateKeys).start();
+
+    // To ensure main thread does not die before outFlux subscribe gets called
+    inducingSleepTime(5000);
+  }
+
+  /**
+   * Unsubscribe using subscription id.
+   *
+   * @param client DaprPreviewClient object
+   */
+  public static void unsubscribeToConfigurationItems(DaprPreviewClient client) {

```

Comment:
```suggestion
  public static void unsubscribeConfigurationItems(DaprPreviewClient client) {

```

and add code snippet to readme also.